tests/sentry/auth/authenticators/test_sms.py

```diff
@@ -0,0 +1,55 @@
+import responses
+from django.http import HttpRequest
+
+from sentry.auth.authenticators import SmsInterface
+from sentry.testutils import TestCase
+
+
+class SmsInterfaceTest(TestCase):
+    def setUp(self):
+        self.user = self.create_user(email="test@example.com", is_superuser=False)
+
+    @responses.activate
+    def test_activate(self):
+        request = HttpRequest()
+        request.user = self.user
+        request.META["REMOTE_ADDR"] = "127.0.0.1"
+
+        responses.add(
```

Comment:
ideally we'd validate outbound payload but i was just trying to make sure the function didnt error